packages/trpc/server/routers/viewer/organizations/list.handler.ts

```diff
@@ -13,7 +11,7 @@ type ListHandlerInput = {
 // This functionality is essentially the same as the teams/list.handler.ts but it's easier for SOC to have it in a separate file.
 export const listHandler = async ({ ctx }: ListHandlerInput) => {
   if (!ctx.user.organization?.id) {
-    throw new TRPCError({ code: "BAD_REQUEST", message: "You do not belong to an organization" });
+    return null;
   }
```

Comment:
Changed this from throwing as there are cases where we need to detect if there is an org or not without using context